src/app/app.component.ts

```diff
@@ -14,17 +17,18 @@ export class MyApp {
 
   rootPage: any = LoginPage;
 
-  pages: Array<{title: string, component: any}>;
+  items: Array<{title: string, component: any}>;
```

Comment:
"items" is not a very meaningful name. pages or components may be better in my opinion.

Reply:
corrected in the patch